src/test/java/org/jabref/logic/importer/fetcher/IacrEprintFetcherTest.java

```diff
@@ -0,0 +1,119 @@
+package org.jabref.logic.importer.fetcher;
+
+import java.util.Optional;
+
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.logic.importer.ImportFormatPreferences;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.BiblatexEntryTypes;
+import org.jabref.model.entry.FieldName;
+import org.jabref.testutils.category.FetcherTest;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockito.Answers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+
+@FetcherTest
+public class IacrEprintFetcherTest {
+
+    private static IacrEprintFetcher fetcher;
+    private static BibEntry abram2017;
+    private static BibEntry beierle2016;
+    private static BibEntry delgado2017;
+
+    @BeforeAll
+    public static void setUp() {
+        fetcher = new IacrEprintFetcher(mock(ImportFormatPreferences.class, Answers.RETURNS_DEEP_STUBS));
+
+        abram2017 = new BibEntry();
+        abram2017.setType(BiblatexEntryTypes.MISC);
+        abram2017.setField("bibtexkey", "cryptoeprint:2017:1118");
+        abram2017.setField(FieldName.ABSTRACT, "The decentralized cryptocurrency Bitcoin has experienced great success but also encountered many challenges. One of the challenges has been the long confirmation time. Another challenge is the lack of incentives at certain steps of the protocol, raising concerns for transaction withholding, selfish mining, etc. To address these challenges, we propose Solida, a decentralized blockchain protocol based on reconfigurable Byzantine consensus augmented by proof-of-work. Solida improves on Bitcoin in confirmation time,  and provides safety  and liveness assuming the adversary control less than (roughly) one-third of the total mining power.\n");
```

Comment:
Please no abstracts in tests, as they are usually subject to copyright of the publisher

Reply:
I'll look into the license details - not having the abstract there is a bit of a problem as fetching the abstract is part of the functionality; therefore it should be tested.
On fist glance, the whole article is published under CC-BY or CC-BY-NC; therefore this shouldn't be a problem as the authors are mentioned right next to the abstract text.
But if you still don't want the abstracts, I can modify the tests accordingly.

Reply:
Do you have a published article that can be received via IACR? In this case you hold the copyright and there is no problem.

Reply:
Not yet ;) But then I'll just remove the abstracts for now and maybe just check that they are not empty.